tools/ipo/ipo4xcmsSet.xml

```diff
@@ -0,0 +1,382 @@
+<tool id="ipo4xcmsSet" name="IPO for xcmsSet" version="0.0.3">
+
+    <description>IPO optimization process for xcms.xcmsSet</description>
+
+    <macros>
+        <import>macros.xml</import>
+
+        <macro name="centwave_ppm_fixed">
+            <param name="ppm" type="integer" value="25" label="Max tolerated ppm m/z deviation in consecutive scans in ppm" help="[ppm]" />
+        </macro>
+        <macro name="centwave_peakwidth_fixed">
+            <param name="min_peakwidth" type="float" value="20" label="Min peak width range in seconds" optional="true" help="[min_peakwidth]" />
+            <param name="max_peakwidth" type="float" value="50" label="Max peak width range in seconds" optional="true" help="[max_peakwidth]" />
+        </macro>
+        <macro name="centwave_mzdiff_fixed">
+            <param name="mzdiff" type="float" value="-0.001" label="Minimum difference in m/z for peaks with overlapping Retention Times" help="[mzdiff] Can be negative to allow overlap" />
+        </macro>
+
+        <macro name="matchedfilter_fwhm_fixed">
+            <param name="fwhm" type="integer" value="30" label="Full width at half maximum of matched filtration gaussian model peak" help="[fwhm] Only used to calculate the actual sigma" />
+        </macro>
+        <macro name="matchedfilter_mzdiff_fixed">
+            <param name="mzdiff" type="float" value="0.6" label="Minimum difference in m/z for peaks with overlapping Retention Times" help="[mzdiff] By default: 0.8-step*steps " />
+        </macro>
+    </macros>
+
+    <expand macro="requirements"/>
+    <expand macro="stdio"/>
+
+    <command><![CDATA[
+        LANG=C Rscript $__tool_directory__/ipo4xcmsSet.r
```

Comment:
quote and remove Rscript ...

Reply:
Quote done
But why removing Rscript.
If you mean LANG, I don't know...